docker-local-mojaloop/postman/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM node:10.15.3-alpine
```

Comment:
I like having the postman repo here for convenience... although I think perhaps we should find a better way to manage it in the future, as this isn't under source control of the original repo anymore.

Reply:
Yea...definitely agree. Lets tackle this down the road.